updatehub/src/main.rs

```diff
@@ -100,7 +104,10 @@ async fn server_main(cmd: ServerOptions) -> updatehub::Result<()> {
     updatehub::logger::init(cmd.verbosity.unwrap_or(slog::Level::Info));
     info!("starting UpdateHub Agent {}", updatehub::version());
 
-    let settings = updatehub::Settings::load()?;
+    let settings = match cmd.conf_file {
+        Some(conf_file) => updatehub::Settings::load(conf_file)?,
+        None => updatehub::Settings::load(PathBuf::from("/etc/updatehub.conf"))?,
+    };
```

Comment:
Use `Option::unwrap_or` here for the `conf_file` identifier. That way we can have a single call to the `Settings::load` function.